plugins/wasi_nn/wasinnenv.h

```diff
@@ -90,6 +109,14 @@ class Graph {
 #ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_TORCH
   torch::jit::Module TorchModel;
 #endif
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_TF
+  TF_Buffer *TFBuffer = nullptr;
+  TF_Graph *TFGraph = nullptr;
+
+  std::filesystem::path TFSavedModelPath;
```

Comment:
Is it possible to set load the saved model in `wasi_nn::load` function and delete the temp dir, so that the saved model path is no longer to be saved here?

Reply:
I plan to place the deletetion of temp dir in the graph class's deconstruction function.